src/autorest-core/lib/autorest-core.ts

```diff
@@ -167,9 +167,14 @@ export class AutoRest extends EventEmitter {
         }
 
         if (view.InputFileUris.length === 0) {
-          throw new Exception("No input files provided.\n\nUse --help to get help information.", 0);
+          if (view.GetEntry("allow-no-input")) {
```

Comment:
we do, the thing is that in said cases the input is infact valid, but autorest currently complains, which causes roughly 2 people per week to ask us why their scripts are "failing" when in fact they are not, AutoRest is just complaining mistakenly